CONTRIBUTING.md

```diff
@@ -164,6 +186,51 @@ After you get your container up and running, you can test it by adding test asse
 
 Finally, commit your changes and submit a PR - we'll take a look at it, provide any needed feedback, and then merge it in. We appreciate any and all feedback!
 
+### Adding a Database Definition to an existing Container
+
+VS Code Dev Containers allow fairly straightforward modification for allowing multiple containers to run together. This is useful when adding database support to an existing container, and can be leveraged with `docker compose.`
+
+1. Create a new copy of an existing language definition folder, and hyphenate the database name.
+   1. For example, `java` becomes `java-postgres`.
+2. Create a new `docker-compose.yml` file.
+   1. Add the existing `Dockerfile` to the `docker-compose` as an `app`. This will be the Dev Container.
+   2. Add the definition for database to the `docker-compose`, version tagging it or using the `latest`. We don't use custom images for databases, as this is generally not needed.
+   3. It will likely be necessary to network the containers so that the `app` can connect to the `db`. This can normally be done by adding the following to the `app` definition:
```

Comment:
```suggestion
   3. It's common that containers will need to be connected through a network so that the `app` can connect to the `db`. This can normally be done by adding the following to the `app` definition:
```